server/controllers/recipe-reaction-controller.js

```diff
@@ -0,0 +1,73 @@
+import db from '../models/index';
+import { successResponse, errorResponse } from '../utils/helpers';
+
+const { Recipe, RecipeRating } = db;
+
+/**
+ *
+ *
+ * @class RecipeReactionController
+ */
+class RecipeReactionController {
+  /**
+   *
+   *@description Rate a recipe functionality
+   * @static
+   * @param {*} req
+   * @param {*} res
+   * @memberof RecipeReactionController
+   * @returns {object} response
+   */
+  static async rating(req, res) {
+    const { id } = req.user;
+    const { slug } = req.params;
+    const { score } = req.body;
+    const validScore = score <= 5 && score >= 1;
```

Comment:
What if I pass a score of 3.567917857? 🙂

Reply:
if a decimal number is used,  a validation error will return a message "must be an integer".